src/server/src/main/resources/db/cassandra/018_sidecar_mode.cql

```diff
@@ -0,0 +1,33 @@
+--
+-- Metrics table for the sidecar mode
+--
+
+CREATE TABLE IF NOT EXISTS node_metrics_v2 (
+    cluster text, 
+    metric_domain text,
+    metric_type text,
+    time_bucket text,
+    host text,
+    metric_scope text,
+    metric_name text,
+    ts timestamp,
+    metric_attribute text,
+    value double,
+    PRIMARY KEY ((cluster, metric_domain, metric_type, time_bucket), host, metric_scope, metric_name, ts, metric_attribute)
```

Comment:
doesn't the `ts` column need to be the first clustering key?

Reply:
Since I have time buckets, I'm not using `ts` in the read queries. This clustering also allows me to get either a specific node or all nodes in the cluster in a single query.